pkg/api/mlflow/api/response/metric.go

```diff
@@ -1,7 +1,10 @@
 package response
 
+import "github.com/G-Research/fasttrack/pkg/database"
+
 // MetricPartialResponse is a partial response object for GetMetricHistoryResponse.
 type MetricPartialResponse struct {
+	RunID     string `json:"run_id"`
```

Comment:
This is not part of the MLFlow protocol. See https://github.com/mlflow/mlflow/blob/v2.3.1/mlflow/protos/service.proto#L456-L469